### PR TITLE
Editorial: Fix `AggregateError` links and description

### PR DESCRIPTION
**Pull request by ExE-Boss**
## See also:
- <https://github.com/tc39/proposal-explicit-resource-management/pull/48>

### CODE REVIEW

**Review by chicoxyzzy — APPROVED** [review]

**Comment by chicoxyzzy** [comment]
Description part looks correct, not sure about links part

File: spec.html [hunk 1]
```diff
@@ -99,19 +98,21 @@ <h1>_NativeError_ Object Structure</h1>
 
     <emu-clause id="sec-properties-of-nativeerror-instances">
       <h1>Properties of _NativeError_ Instances</h1>
-      <p>_NativeError_ instances are ordinary objects that inherit properties from their _NativeError_ prototype object and have an [[ErrorData]] internal slot whose value is *undefined*. The only specified use of [[ErrorData]] is by `Object.prototype.toString` (<emu-xref href="#sec-object.prototype.tostring"></emu-xref>) to identify Error, _NativeError_, or _AggregateError_ instances.</p>
+      <p>_NativeError_ instances are ordinary objects that inherit properties from their _NativeError_ prototype object and have an [[ErrorData]] internal slot whose value is *undefined*. The only specified use of [[ErrorData]] is by `Object.prototype.toString` (<emu-xref href="#sec-object.prototype.tostring"></emu-xref>) to identify Error<ins>, AggregateError</ins> or _NativeError_ instances.</p>
```
Comment (chicoxyzzy) [inline spec.html]: `<ins>` should be placed after comma

File: spec.html [hunk 2]
```diff
@@ -124,55 +125,58 @@ <h1>AggregateError ( _errors_, _message_ )</h1>
           1. Set _O_.[[AggregateErrors]] to _errorsList_.
           1. If _message_ is not _undefined_, then
             1. Let msg be ? ToString(_message_).
-            1. Perform ! CreateMethodProperty(_O_, `"message"`, _msg_).
+            1. Let _msgDesc_ be the PropertyDescriptor { [[Value]]: _msg_, [[Writable]]: *true*, [[Enumerable]]: *false*, [[Configurable]]: *true* }.
```
Comment (chicoxyzzy) [inline spec.html]: Rebase please

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]

File: spec.html (hunk 1 above)
Comment (ExE-Boss) [reply spec.html]: The comma is not present in <https://tc39.es/ecma262/#sec-properties-of-nativeerror-instances>.

File: spec.html [hunk 3]
```diff
@@ -86,11 +85,16 @@ <h1>Well-Known Intrinsic Objects</h1>
...
+  <p>When an ECMAScript implementation detects a runtime error, it throws a new instance of one of the _NativeError_ objects defined in <emu-xref href="#sec-native-error-types-used-in-this-standard"></emu-xref> or a new instance of AggregateError object defined in <emu-xref href="#sec-aggregate-error-object-structure"></emu-xref>. Each of these objects has the structure described below, differing only in the name used as the constructor name instead of _NativeError_, in the `name` property of the prototype object, in the implementation-defined `message` property of the prototype object, and in the presence of the %AggregateError%-specific `errors` property.</p>
```
Comment (mathiasbynens) [inline spec.html:88]: I don't understand these changes. It seems weird to use `_NativeError_` (wrapped in underscores) but `AggregateError (no underscores). Could you elaborate?

Comment (ExE-Boss) [reply spec.html:88]: The underscores mean `<var>`, which means that `_NativeError_` is a template type, whereas `AggregateError` is a concrete type.

Comment (mathiasbynens) [reply spec.html:88]: Thanks!

**Review by mathiasbynens — APPROVED** [review]

**Review by chicoxyzzy — APPROVED** [review]

**Pull request merged**